### PR TITLE
Release 0.6.0

### PR DESCRIPTION
**Pull request by pengyin-shan**
This PR is for the front-end 0.6.0 release:

1. It includes all new updates from the development branch: Title 1 page and components; EQIP API update; $ sign fixes for map tips;

2. Updated CHANGELOG, package.json and package-lock.json (compared to the development branch, this PR only updates these three files)

### CODE REVIEW

**Review by ywkim312 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.6.0] - 2023-06-08
```
Comment (sandeep-ps) [inline CHANGELOG.md:7]: I'm just adding a note that the release date will need to update later.

Comment (pengyin-shan) [reply CHANGELOG.md:7]: Ah yes, thanks for the reminder. Once we determine the final day of release, I'll update this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.6.0] - 2023-06-17
```
Comment (sandeep-ps) [inline CHANGELOG.md:7]:
```suggestion
## [0.6.0] - 2023-07-17
```

Comment (pengyin-shan) [reply CHANGELOG.md:7]: Ops thanks @sandeep-ps !

**Review by sandeep-ps — APPROVED** [review]
Approving. Thanks.

**Review by navarroc — APPROVED** [review]

**Pull request merged**